src/Render.cc

```diff
@@ -14,9 +14,9 @@ using namespace snowcrash;
 
 namespace drafter {
 
-    RenderFormat findRenderFormat(std::string contentType) {
+    RenderFormat findRenderFormat(std::string& contentType) {
 
-        if (contentType == "application/json") {
+        if (contentType.find("application/json") != std::string::npos) {
```

Comment:
I don't think we should do this right now.

Anyway, we might want to do some custom rendering for `application/hal+json`